Whoaverse/Whoaverse/Scripts/whoaverse.js

```diff
@@ -204,7 +204,119 @@ $(document).ready(function () {
             }
         });
     });
-
+    
+    var key = {};
+
+    // List of the inputs the user use when creating a new discussion
+    var submitInputs = {
+      post: [
+        '#Title',
+        '#Subverse',
+        '#MessageContent'
+      ],
+      reply: [
+        '#CommentContent'
+      ]
+    };
+    
+    document.onkeydown = function(evt) {
```

Comment:
@quantumwannabe using a global listener vs 4 listeners might have been a "ram usage" decision. One event looking for *all* eats way less than four elements looking for themselves for the same event to fire the same function.